static/app/utils/advancedAnalytics.tsx

```diff
@@ -43,61 +45,27 @@ export function trackAdvancedAnalyticsEvent<T extends AnalyticsKey>(
   eventKey: T,
   analyticsParams: EventParameters[T],
   org: Organization | null, // if org is undefined, event won't be recorded
-  options?: {startSession: boolean},
-  mapValuesFn?: (params: object) => object
+  options?: Parameters<Hooks['analytics:track-event-v2']>[1]
 ) {
-  try {
-    const {startSession} = options || {};
-    let sessionId = startSession ? startAnalyticsSession() : getAnalyticsSessionId();
-
-    const eventName = allEventMap[eventKey];
-
-    // we should always have a session id but if we don't, we should generate one
-    if (hasAnalyticsDebug() && !sessionId) {
-      // eslint-disable-next-line no-console
-      console.warn(`analytics_session_id absent from event ${eventKey}`);
-      sessionId = startAnalyticsSession();
-    }
-
-    let custom_referrer: string | undefined;
+  const eventName = allEventMap[eventKey];
 
-    try {
-      // pull the referrer from the query parameter of the page
-      const {referrer} = qs.parse(window.location.search) || {};
-      if (typeof referrer === 'string') {
-        // Amplitude has its own referrer which inteferes with our custom referrer
-        custom_referrer = referrer;
-      }
-    } catch {
-      // ignore if this fails to parse
-      // this can happen if we have an invalid query string
-      // e.g. unencoded "%"
-    }
+  const params = {
+    eventKey,
+    eventName,
+    organization: org,
+    ...analyticsParams,
+  };
 
-    // if org is null, we want organization_id to be null
-    const organization_id = org ? org.id : org;
-
-    let params = {
-      eventKey,
-      eventName,
-      analytics_session_id: sessionId,
-      organization_id,
-      role: org?.role,
-      custom_referrer,
-      ...analyticsParams,
-    };
-    if (mapValuesFn) {
-      params = mapValuesFn(params) as any;
-    }
-
-    // could put this into a debug method or for the main trackAnalyticsEvent event
-    if (hasAnalyticsDebug()) {
-      // eslint-disable-next-line no-console
-      console.log('trackAdvancedAnalytics', params);
-    }
-    trackAnalyticsEvent(params);
-  } catch (e) {
+  // could put this into a debug method or for the main trackAnalyticsEvent event
+  if (hasAnalyticsDebug()) {
```

Comment:
Doesn't the getsentry implementation also have debug logging?

Reply:
@markstory yep, but what if you're not using getsentry? It's controlled by a different local storage variable.